pkg/loadbalancer/reconciler/bpf_reconciler_test.go

```diff
@@ -1113,6 +1147,10 @@ func TestBPFOps(t *testing.T) {
 					}
 				}
 
+				if testCase.name == "ClusterIP_add_backends_with_cluster_id" {
+					_ = frontend
+				}
+
```

Comment:
Is this doing something or it is a leftover somehow?

Reply:
Totally forgot about removing that testing leftover 🤦. Thanks!